07-pse/01-pse-07.checkpoint.md

```diff
@@ -0,0 +1,192 @@
+# PSE 7 - Matrix Diagonal Sum
+
+## Given this problem prompt:
+
+Given a square matrix mat, return the sum of the matrix diagonals.
```

Comment:
Should it be `square matrix mat`? Legitimately don't know if that's a common term that I don't know :P

```suggestion
Given a square matrix, return the sum of the matrix diagonals.
```